zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/security/config/ObjectManagementHttpSecurityConfigurer.kt

```diff
@@ -39,6 +39,8 @@ class ObjectManagementHttpSecurityConfigurer : HttpSecurityConfigurer {
                 .hasAuthority(AuthoritiesConstants.ADMIN)
                 .antMatchers(HttpMethod.GET, "/api/v1/object/management/configuration/{id}/object")
                 .hasAuthority(AuthoritiesConstants.ADMIN)
+                .antMatchers(HttpMethod.POST, "/api/v1/object/management/configuration/{id}/object")
+                .hasAuthority(AuthoritiesConstants.ADMIN)
```

Comment:
Every user is allowed to see the objects. So this endpoint should both the role_user